### PR TITLE
188300239 Fix joining with a new table

### PR DESCRIPTION
**Pull request by tealefristoe**
PT Story: https://www.pivotaltracker.com/story/show/188300239

A minor stylistic change I made in another recent PR caused a bug. Lesson learned: change as little as necessary. Especially if your project has no automated tests.

### CODE REVIEW

File: src/ui-pages/join-without-merging.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const JoinWithoutMerging = (props: JoinWithoutMergingProps) => {
         </button>
         <button
           disabled={!joinShareId || (!personalDataLabel && !lastPersonalDataLabel)}
-          onClick={joinShare}>
+          onClick={() => joinShare()}>
```
Comment (emcelroy) [inline src/ui-pages/join-without-merging.tsx:37]: Would it be a little better to not use the anonymous function and keep this as `onClick={joinShare}`?

In a [previous change](https://github.com/concord-consortium/codap-shared-table-plugin/pull/101/files), it looks like you added the anonymous function so you could pass an argument. But you're no longer passing an argument here, so maybe it's not necessary?

**Review by emcelroy — APPROVED** [review]
Looks good 👍 I left a comment/question about a possible small change, but I don't think it's super important. And it's possible I'm missing a good reason why you made that choice.

File: src/ui-pages/join-without-merging.tsx (hunk 1 above)
Comment (tealefristoe) [reply src/ui-pages/join-without-merging.tsx:37]: I was quite surprised, but the change from `() => joinShare()` to `joinShare` is what caused the bug. I now realize that when it's just `joinShare`, the click event gets passed to `joinShare` as an argument, which the function definitely wasn't prepared to handle.

Comment (emcelroy) [reply src/ui-pages/join-without-merging.tsx:37]: I see. Good to know.

**Pull request merged**